ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -321,4 +321,8 @@ if test "${ret}" -eq 0 ; then
   echo "https://$(env KUBECONFIG=${dir}/auth/kubeconfig oc -n openshift-console get routes console -o=jsonpath='{.spec.host}')" > "${SHARED_DIR}/console.url"
 fi
 
+echo "install-config.yaml"
+echo "-------------------"
+cat ${SHARED_DIR}/install-config.yaml | grep -v "password\|username"
```

Comment:
I'd exclude pullSecret as well. The prow censorer will get it, but better safe than sorry.

Reply:
I'd also maybe save it in the artifacts instead of the log? It'd be easier to find that way

Reply:
I just copied this snipped from another file doing the same, so pullSecret was not included there (and one other place)
either. Here's a [PR](https://github.com/openshift/release/pull/31270) to add that to those two places.

I piped this to tee so that we get it as a file in the artifacts as well as the output. overkill?